### PR TITLE
adjust guides naming

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/253.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

**Will backport upon approval.**

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
@mesellings @conceptualshark I'm just playing around with ideas in this PR for effective Guides naming. Do you have any thoughts, suggestions, opinions?

File: docs/guides/configuring-out-of-the-box-connector.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: configuring-out-of-the-box-connectors
-title: Configure an out-of-the-box Connector
+title: Integrate out-of-the-box Connectors
```
Comment (mesellings) [inline docs/guides/configuring-out-of-the-box-connector.md:3]:
```suggestion
title: Integrate a Camunda Connector
```
Suggestion - think we might be moving to using this in place of OOTB as a term with the new Connectors work?

**Review by mesellings — APPROVED** [review]
I like these changes a lot - they make perfect sense - just one note about the Connectors naming, otherwise great work! 🚀

**Comment by christinaausley** [comment]
Thank you for the review @mesellings! Backported 👍

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🥇

File: docs/guides/improve-processes-with-optimize.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: improve-processes-with-optimize
 title: Improve processes with Optimize
-sidebar_label: Improve processes with Optimize
```
Comment (akeller) [inline docs/guides/improve-processes-with-optimize.md:4]: Non-blocking - wasn't "improve" a keyword associated with Optimize? Would you want to adjust the page title, too?

Comment (christinaausley) [reply docs/guides/improve-processes-with-optimize.md:4]: CC @RomanJRW @felix-mueller Do you have opinions here on keywords used with Optimize?

Comment (RomanJRW) [reply docs/guides/improve-processes-with-optimize.md:4]: Not from my side, I think its a PM question predominantly

Comment (christinaausley) [reply docs/guides/improve-processes-with-optimize.md:4]: Merging this PR but following up in ask-optimize on preferred keywords 👍

**Pull request merged**